### PR TITLE
Turn off directpassthrough mode.

### PR DESCRIPTION
**Pull request by jamilatta**
Fixed: #3 

**Relation Issue**: 
https://github.com/closeio/Flask-gzip/issues/7

### CODE REVIEW

File: flask_htmlmin.py [hunk 1]
```diff
@@ -20,6 +20,7 @@ def response_minify(response):
         """
         minify response html to decrease traffic
         """
+        response.direct_passthrough = False
         if response.content_type == u'text/html; charset=utf-8':
```
Comment (hamidfzm) [inline flask_htmlmin.py]: I think it's better to use `response.direct_passthrough = False` after if condition.

Comment (hamidfzm) [inline flask_htmlmin.py]: Just swap the lines and i'll merge this pull request. Thanks.

**Comment by jamilatta** [comment]
Yes, this better after **if** condition, Thanks.

**Comment by jamilatta** [comment]
I`ll do it.

**Comment by jamilatta** [comment]
Done.

**Pull request merged**